PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -4291,7 +4293,12 @@ void Finder::finishFilesSearch(int count, int searchedCount, bool isMatchLines,
 	addSearchHitCount(count, searchedCount, isMatchLines, searchedEntireNotSelection);
 	_scintView.execute(SCI_SETSEL, 0, 0);
 
-	_scintView.execute(SCI_SETLEXER, SCLEX_SEARCHRESULT);

```

Comment:
So the above code was not necessary at all for Scintilla446?
Or Scintilla5 doesn't need to set it again?